app/src/main/java/com/mstudio/superheroarch/Character.kt

```diff
@@ -1,9 +1,24 @@
 package com.mstudio.superheroarch
 
+import android.database.Observable
+import java.util.Objects
+
 data class Character(
     val name: String,
     val status: String,
-    val image: String
+    val image: String,
+    val origin: CharacterOrigin,
```

Comment:
What's the difference between CharacterOrigin and CharacterLocation?

Reply:
They are 2 different objects in the API response, they contain the same, a url and a location, but the values of the origin and the location can vary